hydra-cluster/src/CardanoClient.hs

```diff
@@ -158,15 +88,6 @@ sign signingKey body =
     [makeShelleyKeyWitness body (WitnessPaymentKey signingKey)]
     body
 
--- | Submit a (signed) transaction to the node.
---
--- Throws 'CardanoClientException' if submission fails.
-submit :: NetworkId -> FilePath -> Tx -> IO ()
-submit networkId socket tx =
-  submitTxToNodeLocal (localNodeConnectInfo networkId socket) (TxInMode tx BabbageEraInCardanoMode) >>= \case
-    SubmitSuccess -> pure ()
-    SubmitFail err -> throwIO $ ClientSubmitException{reason = show err, tx}
```

Comment:
Moved to `hydra-node` and renamed as `submitTransaction`

Cf discussion in https://github.com/input-output-hk/hydra-poc/discussions/453